Source/FindInBlueprintExternal/Private/FiBCommandlet.cpp

```diff
@@ -28,6 +31,16 @@ int32 UFiBCommandlet::Main(const FString& Params)
 		StreamSearch.GetFilteredItems(OutItemsFound);
 	}
 
+	const FString OutputFilePath = FPaths::LaunchDir() / TEXT("SearchResults.json");
+	const TUniquePtr<FArchive> FileArchive = TUniquePtr<FArchive>(IFileManager::Get().CreateFileWriter(*OutputFilePath));
+	if (!FileArchive)
+	{
+		UE_LOG(LogFiBCommandlet, Error, TEXT("Failed to create a file writer for the file at location: [%s]."), *OutputFilePath);
+		return 1;
+	}
+
+	FiBResultsToJsonWriter Writer(*FileArchive.Get());
```

Comment:
TUniquePtr / TSharedPtr have overloaded dereference operators ( `*` and `->` ) that will return the object pointed to without needing to call `Get()` first
```suggestion
	FiBResultsToJsonWriter Writer(*FileArchive);
```